server/src/domain/asset/asset.service.ts

```diff
@@ -465,7 +465,15 @@ export class AssetService {
   async handleAssetDeletion(job: IAssetDeletionJob) {
     const { id, fromExternal } = job;
 
-    const asset = await this.assetRepository.getById(id);
+    const asset = await this.assetRepository.getById(id, {
```

Comment:
@danieldietzler  @jrasm91 I've been trying to debug failed tests due to manually adding the relation in this repository call. Any thoughts from your end?

Reply:
Why are you adding all these relations? It looks like only exif would be needed.

Reply:
This is the logic of that method.

![image](https://github.com/immich-app/immich/assets/27055614/c4ef1c0a-6901-4184-8221-efe7450dc6bd)

So I refactored it a little bit to be like this.

![image](https://github.com/immich-app/immich/assets/27055614/31a2e96a-1126-4535-bb5e-ed2efbce793a)

But even then, with just additional exIf relation, the tests failed the same way

Reply:
My rationale for having the `if (!relations)` block was to make future uses of `getById` specify the relations it needs without breaking older code. The refactored code can include unneeded relations if we don't explicitly opt out of them.

Reply:
Ah, forgot to add the additional relation clause to the test call

Reply:
That is quite different behavior than what was there before.